sessionctx/variable/sysvar.go

```diff
@@ -125,6 +125,10 @@ var defaultSysVars = []*SysVar{
 		s.enforceMPPExecution = TiDBOptOn(val)
 		return nil
 	}},
+	{Scope: ScopeSession, Name: TiFlashMaxThreads, Type: TypeInt, Value: strconv.Itoa(config.GetGlobalConfig().Performance.TiFlashMaxThreads), MinValue: -1, MaxValue: math.MaxInt64, SetSession: func(s *SessionVars, val string) error {
+		s.TiflashMaxThreads = TidbOptInt64(val, DefTiFlashMaxThreads)
+		return nil
+	}},
```

Comment:
For MaxValue, can it be `MaxConfigurableConcurrency` (256)? We have previously had misconfiguration problems with allowing thread counts to be very large values.

Reply:
> For MaxValue, can it be `MaxConfigurableConcurrency` (256)? We have previously had misconfiguration problems with allowing thread counts to be very large values.

No problem.